### DISCUSSION TITLE
Name of the File Icons Used in the SideBar of this Image.

### DISCUSSION THREAD

**Discussion by ajimae** [arcticicestudio/nord#248] [Help & Support] [answered]
## Description

This is just a simple question. Can we get the name of the file icon installed for this team as can be seen in the official image.

### Benefits

Can help us to easily search for the File Icons using the name as against using Google image searches.

### Possible Drawbacks

None

## Example

![image](https://user-images.githubusercontent.com/32770340/187794848-963b26e7-d225-40fe-86d9-bf158b7fe4ce.png)

## Additional Context

Whoever knows the name of the Icons should help.

## References

See File Icons below 👇🏽 
<img width="228" alt="ui-overview-jsx" src="https://user-images.githubusercontent.com/32770340/187795187-82aa3822-01f1-4719-b586-3d3ed3311e07.png">

## Potential Solution

N/A

**Comment by cody-quinn** [comment, marked as answer]
I saw this image and instantly thought the same thing, if anyone has an answer that'd be great.

Edit: Found it! https://marketplace.visualstudio.com/items?itemName=Equinusocio.vsc-material-theme-icons

**Comment by svengreb** [comment]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
